docs/index.md

```diff
@@ -271,34 +291,68 @@ Follow these steps:
 	Also available is the `vault delete astra/role org_id="<ORG ID>" role="<ROLE NAME>"` command.
+	
+5. Define a default lease time. Example:
+
+	```bash
+	vault write astra/config \
+	org_id=astra_token=AstraCS:YlABLSDOEMpQlrdWoLLJyzAh:8e34d55b6d774a7822ad87df2e502774749dc5549fd7e2bd248af307bee4ca8b \
+	url=https://api.astra.datastax.com logical_name=org_logical_name renewal_time="4h"
+	```
 
-5. For any of the roles, you can use HashiCorp Vault to generate a new Astra DB token. Example:
+6. For any of the roles, you can use HashiCorp Vault to generate a new Astra DB token. In this example, we'll also specify a lease time that overrides the default. Example:
 
 	```bash
-	vault write astra/org/token org_id="<ORG ID>" role_name="<ROLE NAME>"
+	vault write astra/org/token org_id=7e811ca5-bec5-4ef4-be96-dd24d5284e5c \
+	role_name=Admin_Svc_Acct logical_name="My new token" lease_time=10m 
 	```
 
-	**TIP:** You can also apply custom, meaningful metadata to the generated Astra DB token by adding one or more `metadata` parameters. The metadata names and values can be any free-form text that you want. Example:
+	**TIP:** You can apply custom meaningful metadata to the generated Astra DB token by adding one or more `metadata` parameters. The metadata names and values can be any free-form text that you want. Here we'll also specify a lease_time. Example:
 
 	```bash
 	vault write astra/org/token org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" role_name="organization_administrator" \
-	 metadata="user=mrsmart" metadata="purpose=demo"
+	metadata="user=mrsmart" metadata="purpose=demo" lease_time=20m 
 	```
-
+	
 	The command output displays the new token's properties, including:
 
 	* `clientId` (you'll see this value in Astra DB console too)
 	* `generatedOn` (date format)
 	* `metadata` (example: `map[purpose=demo user=mrsmart]`)
 	* `orgId` (its value)
 	* `token` (example: its `AstraCS:<generated-token-id>` value)
+	* `lease_time` (its value)
 
 	With the newly generated token, you can now make calls to Astra DB via its APIs.
+	
+7.  List token by client ID. Example:
+
+	```bash
+	vault read astra/org/token client_id="ZqUojmgGxvfjhzlJYFqSZjyb" 
+	```
+
+8. Renew a lease. In this example, `UuS2JYK9k5Di9dW9Zq4Ip3v1` is the *Lease ID*:
+
+	```bash
+	vault lease renew astra/org/token/UuS2JYK9k5Di9dW9Zq4Ip3v1
+	
+9. View lease detail. Example where again `UuS2JYK9k5Di9dW9Zq4Ip3v1` is the *Lease ID*:
+
+	```bash
+	vault lease lookup astra/org/token/UuS2JYK9k5Di9dW9Zq4Ip3v1
+	```
+	
+10. List all leases. Example:
+
+	```bash
+	vault list sys/leases/lookup/astra/org/token
+	```
 
-6. You can also delete tokens. Example:
+11. Delete tokens based on an Astra DB `org_id` and `role_name`. Example:
 
 	```bash
-	vault delete astra/org/token org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" role_name="organization_administrator"
+	vault delete astra/org/token org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" \
+	role_name="organization_administrator"
```

Comment:
logical_name is required to delete a token.

Reply:
Thanks again - updated in next commit.